system/security/cryptohelper/digisign.cpp

```diff
@@ -206,7 +206,9 @@ class CDigitalSignatureManager : public CSimpleInterfaceOf<IDigitalSignatureMana
 
 
 static IDigitalSignatureManager * dsm = nullptr;
+#if defined(_USE_OPENSSL) && !defined(_WIN32)
```

Comment:
Why is the second condition present? I get that the first is to suppress "unused static variable" errors...

Reply:
It was matching the existing logic on line 220 in the same file.

Reply:
It's the logic on line 239 it needs to match, though that also mentions WIN32 for reasons I don't really follow

Reply:
There are a few more places its referenced as well.
Also the Windows / OSX version of openssh is currently a different version to the Ubuntu one, which is why I would rather push this off to later.